README.md

```diff
@@ -495,4 +495,4 @@ The following is a high-level list of tasks to progress through each stage of th
 [Explicit Resource Management]: https://github.com/tc39/proposal-explicit-resource-management
 [using-void]: https://github.com/tc39/proposal-explicit-resource-management/blob/main/future/using-void-declaration.md
 [Extractors]: https://github.com/tc39/proposal-extractors
-[Pattern Matching]: https://github.com/tc39/proposal-pattern-matching
\ No newline at end of file
+[Pattern Matching]: https://github.com/tc39/proposal-pattern-matching
```

Comment:
I have no idea of what the difference is here, I only edited the other line through the GitHub UI so I had no auto-formatter enabled.